src/ruby/puppetserver-lib/puppet/server/puppet_config.rb

```diff
@@ -29,9 +29,6 @@ def self.initialize_puppet(puppet_config)
       Puppet[:always_retry_plugins] = false
     end
 
-    # Crank Puppet's log level all the way up and just control it via logback.
```

Comment:
Is there code to actually do this controlling that should be ripped out? Or was it sort of automatic?

Reply:
This function is called to initialize Puppet within Puppetserver, but there was an existing function elsewhere that [enabled Puppetserver specific logging](https://github.com/puppetlabs/puppetserver/blob/master/src/ruby/puppetserver-lib/puppet/server/config.rb#L21), I just used that existing hook for setting the log level.

Reply:
I meant code to "control it via logback". Like, was there special filtering logic or something that we no longer need? (I don't really know anything about logback)

Reply:
The :logback log destination works by grabbing the appropriately named logger and [relaying log messages](https://github.com/puppetlabs/puppetserver/blob/master/src/ruby/puppetserver-lib/puppet/server/logger.rb#L13-L23) to that instance. That logger instance itself should be determining if a message logged via `#info` should be dropped or actually logged.

Reply:
Cool thanks

Reply:
As long as adding:

```
<logger name="puppetserver" level="DEBUG"/>
```

To `/etc/puppetlabs/puppetserver/logback.xml` and restarting the puppetserver service still works, then we should be good from a user workflow perspective.